src/services/ibex/webhook-server/routes/on-receive.ts

```diff
@@ -12,7 +21,38 @@ router.post(
     logRequest, 
     async (req: Request, resp: Response) => {
         const { transaction } = req.body
-        const nsResp = await NotificationsService().ibexTxReceived({paymentHash: transaction.invoice.hash})
+        const paymentAmount = transaction.amount
+        const recipientWalletId = transaction.accountId
+        
+        const receiverWallet = await WalletsRepository().findById(recipientWalletId)
+        if (receiverWallet instanceof RepositoryError) {
+            logger.error(receiverWallet, `Failed to fetch wallet with id ${recipientWalletId}`)
+            return resp.sendStatus(500)
+        }
+
+        const recipientAccountId = receiverWallet.accountId
+        const recipientAccount = await AccountsRepository().findById(recipientAccountId)
+        if (recipientAccount instanceof Error) {
+            logger.error(recipientAccount, `Failed to fetch account with id ${recipientAccountId}`)
+            return resp.sendStatus(500)
+        }
+
+        const recipientUser = await UsersRepository().findById(recipientAccount.kratosUserId)
+        if (recipientUser instanceof Error) {
+            logger.error(recipientUser, `Failed to fetch user with kratos id ${recipientAccount.kratosUserId}`)
+            return resp.sendStatus(500)
+        }
+
+        const nsResp = await NotificationsService().lightningTxReceived({
+            recipientAccountId: recipientAccountId,
+            recipientWalletId,
+            paymentAmount: req.body.transaction.amount,
+            displayPaymentAmount: undefined, // DisplayAmount<DisplayCurrency>
```

Comment:
note: I'm setting this to undefined for the time-being because it introduces a lot of baggage. 

When testing, let's see if this is a required field. If it is, we can add it. If not, we can leave as undefined for simplicity.